app/helpers/profiles_helper.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+# Helper methods for managing Profiles
+module ProfilesHelper
+end
```

Comment:
Delete empty helper modules?

Reply:
Naw.

Reply:
I was in Indiana and drove through a town called Gnawbone

Reply:
I'm iffy on the empty helpers. I'm not sure it there's any benefit to deleting them, and I'm more likely to use them if they exist.

Reply:
> I'll eventually need this

Famous last words. :)